src/tactic/omega/nat/main.lean

```diff
@@ -165,6 +166,7 @@ meta def to_form : nat → expr → tactic (form × nat)
 
 meta def prove_lna : tactic expr :=
 do (p,m) ← target >>= to_form 0,
+    trace p,
```

Comment:
I think you left some debugging trace in

Reply:
You're right, fixed